doc: initiate the documentation for Technical Writer (#142)
bonitasoft/bonita-documentation-site: CONTRIBUTING.adoc

== Pull request

> Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is running to check if your pull request follow commit message format.

Reviewer: ```suggestion
Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is run to check if the pull request follows the commit message format.
```

Reviewer 2: For each improvement, the fix must be submitted with a pull request. When a pull request is submitted, an action is run to check if the pull request follows the commit message format.
